test/libsolidity/SolidityParser.cpp

```diff
@@ -254,7 +254,7 @@ BOOST_AUTO_TEST_CASE(natspec_comment_in_function_body)
 			/// fun1 description
 			function fun1(uint256 a) {
 				var b;
-				/// I should not interfere with actual natspec comments
+				// I should not interfere with actual natspec comments (natspec comments on local variables not allowed anymore)
```

Comment:
Maybe we should remove the comments?

Reply:
Maybe then we should have tests that do interfere with actual natspec comments? (or do we already have those?)

Reply:
@aarlt do we have tests like my previous comment?

Reply:
@leonardoalt at least here there will be no interference possible, because it is just forbidden now on local variables. But maybe we can add some natspecs to other expressions within a function body, I'm not sure whether it will make sense. However, I will check that.

Reply:
Ah right